packages/bpk-component-boilerplate/src/BpkBoilerplate.scss

```diff
@@ -0,0 +1,31 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import '~bpk-mixins/index';
+
+.bpk-boilerplate {
+  padding: $bpk-spacing-md;
```

Comment:
I think this can be removed too, maybe just add `display: flex` so it won't complain about empty declarations. The rest of the styles seem like something we will end up removing for most new components

Reply:
Bear in mind that this package serves two purposes:

1) To be useful for us.
2) To show contributors our conventions and how we structure code.

In this case I figured this would be a good way to show contributors how to use our mixins and follow BEM structure.

Reply:
Having had more time to think about this, I think you're right and I should strip this back. If prospective contributors want to see examples of how we do things, they can easily find it from looking at other packages.